robot_descriptions/loaders/mujoco.py

```diff
@@ -17,6 +17,7 @@
 def load_robot_description(
     description_name: str,
     commit: Optional[str] = None,
+    variant: Optional[str] = "scene",
```

Comment:
```suggestion
    variant: Optional[str] = None,
```

I suggest we keep the API backward compatible and only do a minor version bump for the next release. Users can load scenes using this feature at present, and we can address the full feature https://github.com/robot-descriptions/robot_descriptions.py/issues/62 for the next major version.